rollup.config.js

```diff
@@ -9,12 +9,34 @@ export default [
       ...Object.keys(pkg.peerDependencies),
     ],
     output: [
-      { file: pkg.main, format: 'cjs' },
-      { file: pkg.module, format: 'es' },
+      {
+        file: pkg.main,
+        format: 'cjs',
+        plugins: [getBabelOutputPlugin({
+          presets: [['airbnb', {
+            modules: false,
+            runtimeVersion: '7.12.5',
+            runtimeHelpersUseESModules: false,
+          }]],
+        })],
+      },
+      {
+        file: pkg.module,
+        format: 'es',
+        plugins: [getBabelOutputPlugin({
+          presets: [['airbnb', {
+            modules: false,
+            runtimeVersion: '7.12.5',
+            runtimeHelpersUseESModules: true,
+          }]],
+        })],
+      },
     ],
     plugins: [
       babel({
+        babelrc: false,
         babelHelpers: 'bundled',
```

Comment:
Although don't we want to change this to `runtime` instead of using the getBabelOutputPlugin method? I'll try out an alternative approach now.

Reply:
https://github.com/civiccc/react-waypoint/pull/353